src/components/Hoverable/index.js

```diff
@@ -98,7 +98,7 @@ class Hoverable extends Component {
                 onMouseEnter={() => this.setIsHovered(true)}
                 onMouseLeave={() => this.setIsHovered(false)}
                 onBlur={(el) => {
-                    if (this.wrapperView.contains(el.relatedTarget)) {
+                    if (!this.wrapperView || this.wrapperView.contains(el.relatedTarget)) {
                         return;
```

Comment:
Shouldn't this be `&&` ?

Reply:
The original logic is
```javascript
if (!this.wrapperView.contains(el.relatedTarget)) {
     this.setIsHovered(false)
}
```
Adding an empty judgment condition becomes
```javascript
if (this.wrapperView && !this.wrapperView.contains(el.relatedTarget)) {
     this.setIsHovered(false)
}
```
Because here ESLint needs to return in advance, so we need to change `first && second` to `!first || !second`, which will become
```javascript
if (!this.wrapperView || this.wrapperView.contains(el.relatedTarget)) {
     return;
}
```

Reply:
Oops I miss read that.

Reply:
how this change is also relevant to this PR?

Reply:
> Using OR is kind of confusing, this should work right?

@ Santhosh-Sellavel
Sorry, But this doesn't work, it should be
```javascript
if (this. wrapperView && this. wrapperView. contains(el. relatedTarget)) {
     this. setIsHovered(false)
}
```
But ESLint lets us return early, so it would be the opposite


---



> how this change is also relevant to this PR?

https://github.com/Expensify/App/issues/16357#issuecomment-1504866062
I have done a full explanation here

Reply:
@hellohublot This is how it was now without changes from this PR

```
// If wrapperView contains the target we should return here without setting false
if (this.wrapperView.contains(el.relatedTarget)) {
      return;
  }
 this.setIsHovered(false);
```

The problem is `wrapperView` could be `null` or `undefined` causing console errors. As we know only when `wrapperView` contains we should skip setting `this.setIsHovered(false)`. So the following makes sense, what's wrong here?

```js
if (this.wrapperView && this.wrapperView.contains(el.relatedTarget)) {
     return;
}
this.setIsHovered(false);
```

Reply:
` this.wrapperView` is `null` this makes `this.wrapperView.contains(el.relatedTarget)` check `false`, so this.setIsHovered(false) should be executed right? 

Here we skip it means like a bug to me

Reply:
> I guess, this bug should is existed before, but everyone did not see the console error, so the error thrown by the console means that the execution is interrupted, which is consistent with our logic here

If that's the case we should avoid these changes here, maybe start reporting it on expensify-bugs on decide on it how to move on with this.